target/linux/layerscape/image/armv8_64b.mk

```diff
@@ -1,17 +1,19 @@
 #
-# Copyright 2018 NXP
+# Copyright 2018-2020 NXP
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
 
 define Device/Default
   PROFILES := Default
-  IMAGES := firmware.bin
+  IMAGES := firmware.bin sysupgrade.bin
   FILESYSTEMS := squashfs
   KERNEL := kernel-bin | gzip | uImage gzip
   KERNEL_LOADADDR := 0x80080000
   KERNEL_ENTRY_POINT := 0x80080000
+  IMAGE/sysupgrade.bin := append-kernel | pad-to 16M | \
+    append-rootfs | pad-rootfs | check-size 50331649 | append-metadata
```

Comment:
Where is the argument for check-size coming from?
Consider defining IMAGE_SIZE instead.